sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -92,6 +92,24 @@ def wrapper(self, *args, **kwargs):
           frame_base.populate_defaults(pd.DataFrame)(wrapper)))
 
 
+LIFTABLE_AGGREGATIONS = ['all', 'any', 'max', 'min', 'prod', 'sum']
+LIFTABLE_WITH_SUM_AGGREGATIONS = ['size', 'count']
+UNLIFTABLE_AGGREGATIONS = ['mean', 'median', 'std', 'var']
+
+
+def _agg_method(base, func):
+  def wrapper(self, *args, **kwargs):
+    return self.agg(func, *args, **kwargs)
+
+  if func in UNLIFTABLE_AGGREGATIONS:
+    wrapper.__doc__ = (
+        f"``{func}`` cannot currently be parallelized, it will "
```

Comment:
"cannot currently be parallelized, it will" -> "cannot currently be parallelized. It will"

Reply:
Fixed this here and everywhere else that I saw this pattern.